Common subdirectories: fuse-1.0.0.1a-old/compat and fuse-1.0.0.1a-new/compat
Common subdirectories: fuse-1.0.0.1a-old/debugger and fuse-1.0.0.1a-new/debugger
Common subdirectories: fuse-1.0.0.1a-old/disk and fuse-1.0.0.1a-new/disk
Common subdirectories: fuse-1.0.0.1a-old/hacking and fuse-1.0.0.1a-new/hacking
Common subdirectories: fuse-1.0.0.1a-old/ide and fuse-1.0.0.1a-new/ide
Common subdirectories: fuse-1.0.0.1a-old/lib and fuse-1.0.0.1a-new/lib
Common subdirectories: fuse-1.0.0.1a-old/machines and fuse-1.0.0.1a-new/machines
Common subdirectories: fuse-1.0.0.1a-old/man and fuse-1.0.0.1a-new/man
Common subdirectories: fuse-1.0.0.1a-old/perl and fuse-1.0.0.1a-new/perl
Common subdirectories: fuse-1.0.0.1a-old/pokefinder and fuse-1.0.0.1a-new/pokefinder
Common subdirectories: fuse-1.0.0.1a-old/roms and fuse-1.0.0.1a-new/roms
diff -up fuse-1.0.0.1a-old/screenshot.c fuse-1.0.0.1a-new/screenshot.c
--- fuse-1.0.0.1a-old/screenshot.c	2012-02-08 15:50:00.009801232 +0100
+++ fuse-1.0.0.1a-new/screenshot.c	2012-02-08 15:53:19.029248926 +0100
@@ -49,6 +49,7 @@
 #ifdef USE_LIBPNG
 
 #include <png.h>
+#include <zlib.h>
 
 static int get_rgb32_data( libspectrum_byte *rgb32_data, size_t stride,
 			   size_t height, size_t width );
Common subdirectories: fuse-1.0.0.1a-old/sound and fuse-1.0.0.1a-new/sound
Common subdirectories: fuse-1.0.0.1a-old/timer and fuse-1.0.0.1a-new/timer
Common subdirectories: fuse-1.0.0.1a-old/ui and fuse-1.0.0.1a-new/ui
Common subdirectories: fuse-1.0.0.1a-old/unittests and fuse-1.0.0.1a-new/unittests
Common subdirectories: fuse-1.0.0.1a-old/z80 and fuse-1.0.0.1a-new/z80
